config/initializers/breakers.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+require 'rx/configuration'
+
+# Read the redis config, create a connection and a namespace for breakers
+redis_config = Rails.application.config_for(:redis).freeze
```

Comment:
I think it's clearer for it to have its own initializer rather than sticking it into the Redis one, and the cost of creating another Redis connection seems low enough that it's fine.